modules/ztp-sno-du-enabling-workload-partitioning.adoc

```diff
@@ -10,42 +10,37 @@
 
 [NOTE]
 ====
-Workload partitioning can only be enabled during cluster installation. You cannot disable workload partitioning post-installation. However, you can reconfigure workload partitioning by updating the `cpu` value that you define in the performance profile, and in the related `MachineConfig` custom resource (CR).
+Workload partitioning can be enabled during cluster installation only.
+You cannot disable workload partitioning post-installation.
+You can however change the set of CPUs assigned to the isolated and reserved sets through the `PerformanceProfile` CR.
+Changes to CPU settings cause the node to reboot.
 ====
 
-* The base64-encoded CR that enables workload partitioning contains the CPU set that the management workloads are constrained to. Encode host-specific values for `crio.conf` and `kubelet.conf` in base64. Adjust the content to match the CPU set that is specified in the cluster performance profile. It must match the number of cores in the cluster host.
-+
-include::snippets/ztp-02-master-workload-partitioning.adoc[leveloffset=+1]
-
-* When configured in the cluster host, the contents of `/etc/crio/crio.conf.d/01-workload-partitioning` should look like this:
-+
-[source,terminal]
-----
-[crio.runtime.workloads.management]
-activation_annotation = "target.workload.openshift.io/management"
-annotation_prefix = "resources.workload.openshift.io"
-resources = { "cpushares" = 0, "cpuset" = "0-1,52-53" } <1>
-----
-<1> The `cpuset` value varies based on the installation.
-If Hyper-Threading is enabled, specify both threads for each core. The `cpuset` value must match the reserved CPUs that you define in the `spec.cpu.reserved` field in the performance profile.
+.Upgrading from {product-title} 4.12 to 4.13+
+[NOTE]
+====
+When upgrading {ztp} 4.12 to version 4.13 or later, remove the workload partitioning `MachineConfig` CRs from the `/extra-manifest` folder that you use to provision the cluster.
```

Comment:
I think we may need to update this based on the fact that 4.13 is not GA. Maybe something along the lines of "when transitioning to cpuPartitioningMode for enabling WLP remove the ...."